client/commands/retro.js

```diff
@@ -0,0 +1,11 @@
+/* global commandFuncs, openFlexPanel, getServiceBaseURL, GAME */
+
+commandFuncs.retro = args => {
+  if (args.help || args._.length > 1) {
+    return  // handled by game-cli in server/index.js
+  }
+
+  const projectPath = args._.length === 1 ? `/${args._[0].replace('#', '')}` : ''
+  const flexPanelURL = `${getServiceBaseURL(GAME)}/retro${projectPath}#${Date.now()}`
```

Comment:
Nitpick: can we make this `/retro?project={project name}` instead of `/retro/{project name}`?

Feeling just a little bit of tension about paths `/` and `/{name}` returning the same view.